src/client/components/AddNewRoom/RoomHandler.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react';
+
+import AddRoom from './AddRoom';
+
+const RoomHandler = () => {
+  const [newRoom, setNewRoom] = useState('');
+  const [roomId, setRoomId] = useState('');
+  const inputChange = (e) => {
+    setNewRoom(e.target.value);
+  };
+
+  // send data fucntion
+
+  async function postData(data) {
+    const response = await fetch('http://localhost:3000/api/channels', {
```

Comment:
Usually, we don't use full URLs here. Instead, we are using just `/api/channels`